pkg/sql/logictest/testdata/logic_test/enums

```diff
@@ -431,10 +431,17 @@ SELECT * FROM enum_array
 {hello} {hello}
 {howdy} {howdy}
 
-query TTT
-SELECT pg_typeof(x), pg_typeof(x[1]), pg_typeof(ARRAY['hello']::_greeting) FROM enum_array LIMIT 1
-----
-public.greeting[]  public.greeting  public.greeting[]
+# TODO (rohany): Commenting out this test because the types return either public.greeting[] or
```

Comment:
Pinging to see if @jordanlewis or @ajwerner have thoughts here. I'm going to be redoing this work soon